app.js

```diff
@@ -164,6 +164,34 @@ projectBtns.forEach((btn) => {
   });
 });
 
+//  Local Storage
+//  Object to store User Data in Local Storage
+const userData = {
+  name: '',
+  email: '',
+  message: '',
+};
+
+const userName = document.getElementById('name');
+const userEmail = document.getElementById('email');
+const userMessage = document.getElementById('message');
+
+const getData = localStorage.getItem('UserData');
+const newData = JSON.parse(getData);
+userName.value = newData.name;
+userEmail.value = newData.email;
+userMessage.value = newData.message;
+
+// Function to save data in local storage
+function saveData() {
+  userData.name = userName.value;
+  userData.email = userEmail.value;
+  userData.message = userMessage.value;
+  // Convert userData to JSON format
+  const localData = JSON.stringify(userData);
+  localStorage.setItem('UserData', localData);
+}
```

Comment:
- [ ] Kindly fix localStorage it's not recording the value of input fields right now. When you enter a data end click outside of the input box localStorage should save the value for re-reload. You may use blur event listener to give the control to the input fields.
https://screen-recorder-bucket.s3.ap-south-1.amazonaws.com/ScreenRecorder_2022-10-13_acf8cf03-5248-427a-912a-4ebf0ee420b3.mp4